README.md

```diff
@@ -9,6 +9,11 @@ Eclipse Collections
 
 Eclipse Collections is a collections framework for Java. It has JDK-compatible List, Set and Map implementations with a rich API, additional types not found in the JDK like Bags, Multimaps, and set of utility classes that work with any JDK compatible Collections, Arrays, Maps, or Strings. The iteration protocol was inspired by the Smalltalk collection framework.
 
+Learn Eclipse Collections
+------------------------
+Check out the [Eclipse Collections Kata](https://github.com/eclipse/eclipse-collections-kata) that is a fun way to learn Eclipse Collections. [Kata](https://en.wikipedia.org/wiki/Kata) is an exercise in Japanese martial arts concept. [Eclipse Collections Kata](https://github.com/eclipse/eclipse-collections-kata) is set-up as a series of junit tests that fail. Your task is to make them pass using Eclipse Collections, and you get to learn the usage patterns of the framework by going through the exercises. 
+
```

Comment:
Seems like the text and markdown could be closer to the copy in the kata.
https://raw.githubusercontent.com/eclipse/eclipse-collections-kata/master/README.md

```
A [kata](https://en.wikipedia.org/wiki/Kata) is an exercise in martial arts. 
A [code kata](http://codekata.com/) is an exercise in programming which helps hone your skills through practice and repetition. 
The Eclipse Collections Kata is a fun way to help you learn idiomatic Eclipse Collections usage. 
This particular kata is set up as a series of unit tests which fail. 
Your task is to make them pass, using Eclipse Collections.
```

Tweaking it slightly for this repo it could say:
```
Check out the [Eclipse Collections Kata](https://github.com/eclipse/eclipse-collections-kata), a fun way to help you learn idiomatic Eclipse Collections usage. 
A [kata](https://en.wikipedia.org/wiki/Kata) is an exercise in martial arts. 
A [code kata](http://codekata.com/) is an exercise in programming which helps hone your skills through practice and repetition. 
This particular kata is set up as a series of unit tests which fail. 
Your task is to make them pass, using Eclipse Collections.
```